.prettierignore

```diff
@@ -0,0 +1 @@
+**/
```

Comment:
I suggest ignore only the vendors files

Reply:
We do not have a specific vendor folder. All the warnings indicated by prettier are related to files created by us. I have created a ticket to resolve this as soon as possible.
https://makingsense.atlassian.net/browse/DS-4596